extensions/azdata/src/test/azdata.test.ts

```diff
@@ -26,14 +26,14 @@ describe('azdata', function () {
 	});
 
 	describe('findAzdata', function () {
+		sinon.stub(utils, 'searchForCmd').returns(Promise.resolve('C:\\path\\to\\azdata.cmd'));
```

Comment:
The point of this test is to mock out it failing to find the initial exe - by doing this you're testing a different path. Please keep the current tests intact and just stub out the appropriate methods based on the platform the tests are running on.

Reply:
Are you talking about the successful test or the unsuccessful one? For the successful one, I have just rearranged the order in which stubs are created so the test should be same as before.

So assuming your concern is for the unsuccessful one. So for unsuccessful one, there is on searching for exe, so then, keeping the original test is relevant only for win32, the one, I have created is relevant for all platforms. I will add back the original test just for windows so that we will now have 3 tests.

Reply:
fixed